rest-common/src/main/java/uk/ac/ebi/quickgo/rest/search/solr/SolrQueryConverter.java

```
@@ -82,10 +83,54 @@ public SolrQueryConverter(String requestHandler, QueryVisitor<String> solrQueryS
             solrQuery.setParam(FACET_ANALYTICS_ID, aggregateConverter.convert(request.getAggregate()));
         }
 
+        if (!request.getSortCriteria().isEmpty()) {
+            request.getSortCriteria().forEach(criterion ->
+                    solrQuery.addSort(criterion.getSortField().getField(), sortOrderOf(criterion.getSortOrder())));
+        }
+
         return solrQuery;
     }
 
-    private int calculateRowsFromPage(int page, int numRows) {
-        return (page - 1) * numRows;
+    private SolrQuery.ORDER sortOrderOf(SortCriterion.SortOrder sortOrder) {
+        switch (sortOrder) {
+            case ASC:
+                return SolrQuery.ORDER.asc;
+            case DESC:
+                return SolrQuery.ORDER.desc;
+            default:
+                throw new IllegalStateException("Unknown sort order specified");
```

yes, annoying one that. The idea is that if in the future `SortOrder` was extended to include some other values, then this `default` branch would be reached, and the caller (our code) will have their application fail, with a clear message, at development time -- rather than getting a valid value, and probably logging a warning. What's also annoying is that it's not possible to test, unless I redefine `SortOrder` ...

Is it okay with you if I leave the exception in place, so that it is then the caller's responsibility to deal with the issue (e.g., add another switch branch) when they're doing development.